ethereumkit/src/main/java/org/p2p/ethereumkit/external/api/EthereumNetworkModule.kt

```diff
@@ -17,12 +19,14 @@ import java.util.logging.Logger
 
 private const val QUALIFIER_ETH_HTTP_CLIENT = "ethereum"
 
-object EthereumNetworkModule {
+internal object EthereumNetworkModule {
 
     fun create(): Module = module {
         single(named(QUALIFIER_ETH_HTTP_CLIENT)) { getOkHttpClient() }
-        single { getRetrofit(get(named(QUALIFIER_ETH_HTTP_CLIENT)),get()) }
-        single { GsonProvider().provide()}
+        single { getRetrofit(get(named(QUALIFIER_ETH_HTTP_CLIENT)), get()) }
+        single { GsonProvider().provide() }
```

Comment:
better to extract this Gson dependency to some Infrastructure module

Reply:
Provide named scope for this dependency